src/Materials/PBR/pbrBaseMaterial.ts

```diff
@@ -679,7 +679,8 @@ export abstract class PBRBaseMaterial extends PushMaterial {
      * Should this material render to several textures at once
      */
     public get shouldRenderToMRT() {
-        return this.subSurface.isScatteringEnabled;
+        return this.subSurface.isScatteringEnabled ||

```

Comment:
Do you think materialShouldRenderGeometry could be enough so the test is simpler with SSS ?

Reply:
actually with subsurface we need irradiance, and with ssao we need geometry information
Your comment is still valid though, for harmonization I will replace this by a boolean named `materialsShouldRenderIrradiance`